cypress/integration/page.spec.js

```diff
@@ -1026,7 +1104,7 @@
       page.spyOnCommentsPost();
 
       newCommentForm.emailInput().type('some@email.com');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
Consider using a more descriptive comment body for the test input to improve readability.